environment_setup/ruby.sh

```diff
@@ -1,39 +1,9 @@
 #!/usr/bin/env bash
 
+. etc/helpers.sh
```

Comment:
Not sure why I didn't think to do this before haha I blame my novice knowledge of bash 😉